samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -173,152 +251,151 @@ class DailyCallManager {
   }
 
   /**
-   * Decides whether to display video or play audio based on the track type.
-   * @param {Object} event - The track event object.
-   */
-  displayTrack(event) {
-    if (event.track.kind === 'video') {
-      this.displayVideo(event);
-    } else {
-      this.playAudio(event);
-    }
-  }
-
-  /**
-   * Adds a <video> element to the DOM and attaches the video track to it.
-   * Also displays the session ID of the participant as an overlay.
-   * If the video track is stopped, keeps the placeholder.
-   * @param {Object} event - The track event object containing the video track.
+   * Creates and sets up a new video container for a specific participant. This function dynamically
+   * generates a video element along with a container and an overlay displaying the participant's ID.
+   * The newly created elements are appended to a designated parent in the DOM, preparing them for video
+   * streaming or playback related to the specified participant.
+   *
+   * @param {string} participantId - The unique identifier for the participant.
    */
-  displayVideo(event) {
-    const participantId = event.participant.session_id;
-    let videoContainer = document.getElementById(
-      `video-container-${participantId}`
-    );
-
-    // If a container for this participant doesn't exist, create it
-    if (!videoContainer) {
-      videoContainer = document.createElement('div');
-      videoContainer.id = `video-container-${participantId}`; // Use participant ID for unique container ID
-      videoContainer.className = 'video-container';
-      videoContainer.style.backgroundColor = '#ccc'; // Set a gray background for the placeholder
-      document.getElementById('videos').appendChild(videoContainer);
-
-      const sessionIdText = document.createElement('div');
-      sessionIdText.className = 'session-id-overlay';
-      sessionIdText.textContent = participantId;
-      videoContainer.appendChild(sessionIdText);
-    }
-
-    // Check if event track is active or not
-    if (!event.track.enabled) {
-      if (videoContainer.querySelector('video')) {
-        // If the video track is disabled but video element exists, clean up
-        videoContainer.querySelector('video').srcObject = null; // Free up media resources
-        // You might also want to change the placeholder style or add text/icon to indicate camera is off
-      }
-      return; // Exit without further processing if track is not active
-    }
-
-    // Create new video element and add to container if not exists or reuse existing one
-    const videoEl =
-      videoContainer.querySelector('video') || document.createElement('video');
+  createVideoContainer(participantId) {
```

Comment:
I can't comment in `leave()` because it hasn't changed, so commenting here instead... on `leave()` we null out the video source and remove the video element, but shouldn't we remove the whole container, complete with video & overlay? or is that somewhere i just overlooked?

Reply:
I do this in `handleLeave()`:
```
const videosDiv = document.getElementById('videos');
while (videosDiv.firstChild) {
  videosDiv.removeChild(videosDiv.firstChild);
}
```